proposals/lip-0071.md

```diff
@@ -28,7 +28,7 @@ In the current [Lisk protocol][lip-0042], the rewards are the same for all activ
 
 ### Minimum Reward for Active Validators
 
-* This module assigns a minimum reward i.e. `minimumRewardActiveValidators` per round to all active validators to cover the minimal operational cost to run a validator. For the Lisk mainchain, the minimum reward is 10<sup>7</sup> beddows or `0.1 LSK`.
+This module assigns a minimum reward i.e. `minimumRewardActiveValidators` per round to all active validators to cover the minimal operational cost to run a validator. For the Lisk mainchain, the minimum reward is 10<sup>7</sup> beddows or `0.1 LSK`.
```

Comment:
I just noticed that in line 36 we have the phrase "the current validator proof-of-stake (PoS) system", which was edited during the terminology change. Should we remove the word "validator" there? Also not sure if we should call it "current" or "previous".

Reply:
Thanks for pointing out. I have updated the text.